### PR TITLE
Added single signout

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
@clintonb Please review

### CODE REVIEW

**Comment by clintonb** [comment]
👍

**Comment by awais786** [comment]
@ahsan-ul-haq you have to change in the configuration repo too.  Since you have removed the accounts from urls.
[credentials_logout_url](https://github.com/edx/configuration/pull/3123/files#diff-240da08108c4d3cce14b2abd689b924eR51)

FYI @clintonb

File: credentials/apps/api/tests/test_views.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ def test_get(self):
         path = reverse("api:v1:usercredential-detail", args=[self.user_credential.id])
         response = self.client.get(path)
         self.assertEqual(response.status_code, 200)
+        # pylint: disable=no-member
```
Comment (clintonb) [inline credentials/apps/api/tests/test_views.py]: Write this once at the top of the file.

**Comment by clintonb** [comment]
@awais786 nice catch!

**Pull request merged**